Library/Homebrew/test/tap_spec.rb

```diff
@@ -209,6 +209,31 @@ def setup_git_repo
       expect { already_tapped_tap.install full_clone: true }.to raise_error(TapAlreadyUnshallowError)
     end
 
+    describe "force_auto_update" do
+      it "defaults to nil" do
+        setup_git_repo
+        already_tapped_tap = described_class.new("Homebrew", "foo")
```

Comment:
Use `let(:already_tapped_tap)` to define this, since it is used in all three specs.